### ISSUE TITLE
Feature Request: Snippets

### ISSUE THREAD

**Issue by bradleyjones** [Kapeli/Dash-iOS#74]
Would be great to view snippets on the iOS application, especially if you've set the snippet library location on the Mac app to your iCloud Drive.

**Comment by JiveDig** [comment]
This would be extremely useful and is often frustrating when I need snippets and don’t have access to my computer. Love Dash otherwise!

**Comment by lbonillaii** [comment]
Adding Dash as a keyboard to add your snippets would be neat

**Comment by philwhiles** [comment]
I agree - I use Dash on both macOS and iOS, for work and personal use. As I am freelance and always have my iPad at my side at work, I would love to be able to access my macOS stored snippets for reference.

**Comment by jingx23** [comment]
Would be really great to see this coming for iOS. It‘s also useful if your working with playgrounds to copy things out or into the dash snippets and have them synced via iCloud.

**Comment by jonathangroves** [comment]
+1 from me for implementing snippets on iPadOS - perhaps with iCloud sharing. Trying to beef up my iPad as a dev machine and this is the missing element.
Keep safe...

**Comment by tnchuntic** [comment]
Merge multiple snippets or multiple source of snippets.
ex. 

- Office snippets 
- Home snippets
